.changeset/late-bees-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/selectable-search-input': minor
+---
+
+support for an additional action added + minor visual changes
```

Comment:
Support for an optional custom action icon to be displayed on the right side of the selectable search input.